client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -307,7 +307,7 @@ export class List extends React.Component {
 				className="domain-management-list__add-a-domain"
 				onClick={ this.clickAddDomain }
 			>
-				{ this.props.translate( 'Add Domain' ) }
+				{ this.props.translate( 'Add domain' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.